charts/callisto-base-service/templates/ingress.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.ingress }}
```

Comment:
Add a guard to only deploy ingress for branch deployment, as ingress for higher environments is managed in `callisto-ingress-nginx`